src/renderer/components/ParticipantsTile/index.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <section class="tile is-parent">
+    <div class="tile is-child box">
+      <h3 class="title">Meeting participants</h3>
+
+      <participant-list v-bind:list="list"></participant-list>
+      <add-participant :onAdded="onParticipantAdded"></add-participant>
+    </div>
+  </section>
+</template>
+
+<script>
+import AddParticipant from './AddParticipant';
+import ParticipantList from './ParticipantList';
+
+export default {
+  components: { AddParticipant, ParticipantList },
+  props: {
+    list: { type: Array },
+  },
+  methods: {
+    onParticipantAdded(newParticipant) {
+      this.list.push(newParticipant);
```

Comment:
This operation should be done in parent component (the "owner" of participants list) via events. Or ideally in Vuex - the simplest and most readable solution.

Reply:
Next PR will include using vuex

Reply:
I'm waiting for it :)